pkg/defaults/defaults.go

```diff
@@ -408,4 +408,7 @@ const (
 
 	// EnableBPFBypassFIBLookup instructs Cilium to enable the FIB lookup bypass optimization for nodeport reverse NAT handling.
 	EnableBPFBypassFIBLookup = true
+
+	// InstallNoConntrackRules instructs Cilium to install Iptables rules to skip netfilter connection tracking on all pod-to-pod traffic.
+	InstallNoConntrackIptRules = true
```

Comment:
Given the discussion on Slack, I can't see how we can safely enable this by default without risking at least some users just to have non-functional clusters with hard to debug problems. Am I missing something?

The default should be guaranteed to succeed on all platforms.